sql/core/src/test/scala/org/apache/spark/sql/SubquerySuite.scala

```diff
@@ -1371,4 +1371,21 @@ class SubquerySuite extends QueryTest with SharedSQLContext {
       }
     }
   }
+
+  test("SPARK-27782 scalar subquuery name should start with scalar-subquery#") {
```

Comment:
`subquuery` -> `subquery`. And, please remove `SPARK-27782` since this is a new feature instead of bug. We use JIRA id for bugs in general.